grakn-client/src/main/java/ai/grakn/remote/concept/RemoteConcepts.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Grakn - A Distributed Semantic Database
+ * Copyright (C) 2016-2018 Grakn Labs Limited
+ *
+ * Grakn is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Grakn is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Grakn. If not, see <http://www.gnu.org/licenses/gpl.txt>.
+ */
+
+package ai.grakn.remote.concept;
+
+import ai.grakn.concept.Attribute;
+import ai.grakn.concept.AttributeType;
+import ai.grakn.concept.ConceptId;
+import ai.grakn.concept.Entity;
+import ai.grakn.concept.EntityType;
+import ai.grakn.concept.Relationship;
+import ai.grakn.concept.RelationshipType;
+import ai.grakn.concept.Role;
+import ai.grakn.concept.Rule;
+import ai.grakn.concept.Type;
+import ai.grakn.remote.RemoteGraknTx;
+
+/**
+ * @author Felix Chapman
+ */
+public class RemoteConcepts {
```

Comment:
These Java docs make me sad. Also maybe the class should be called `RemoteConceptFactory` ?

Reply:
You're right on the Javadocs! I keep forgetting.

I picked this naming convention because it's what I do in Graql - and the same one used in Guava.